charts/pulsar/templates/bookkeeper-configmap.yaml

```diff
@@ -22,7 +22,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: "{{ template "pulsar.fullname" . }}-{{ .Values.bookkeeper.component }}"
-  namespace: {{ .Values.namespace }}
+  namespace: {{ .Values.namespace | default .Release.Namespace }}
```

Comment:
Can you implement this using a template function? 

Like what we did in https://github.com/streamnative/charts/blob/master/charts/pulsar/templates/_helpers.tpl#L21

Reply:
sure, see updated commit